src/config.rs

```diff
@@ -667,6 +695,16 @@ impl GooseDefaultType<usize> for GooseAttack {
             GooseDefault::ThrottleRequests => self.defaults.throttle_requests = Some(value),
             GooseDefault::TelnetPort => self.defaults.telnet_port = Some(value as u16),
             GooseDefault::WebSocketPort => self.defaults.websocket_port = Some(value as u16),
+            #[cfg(feature = "pdf-reports")]
+            GooseDefault::PdfChromeTimeout => {
+                // Note: We'll need to add this field to GooseDefaults struct
+                // For now, we'll return an error indicating this needs implementation
```

Comment:
Ohhh. I get it. This is just for the telnet controller?

Reply:
Implemented, and fixed name consistency with the command line option. (Chrome is an internal implementation detail that's hidden behind a feature - plus, we needed a shorter change so that it doesn't wrap on the help screen.)